Gopkg.toml

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
Not a blocker, but we might want to dump this big comment in the tomls

Reply:
All I did was `dep init`.